assets/js/home.js

```diff
@@ -0,0 +1,5 @@
+$(function () {
+  $("input[type='image']").on('click', function(){
+    $("input[id='upload']").click();
+  });
```

Comment:
This code does not have enough documentation to help understand what is happening over here.